rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -228,7 +228,7 @@ public static int getFileValidationInterval(Configuration conf)
 
   public static int getClientTimeout(Configuration conf)
```

Comment:
Change the CacheConfig method names to reflect the config option changes.

Reply:
Done